src/apis/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+export const PROFILE_DATA = async () => {
+  try {
+    const response = await axios.get(
+      "https://bootcamp-api.codeit.kr/api/sample/user"
+    );
+    return response;
+  } catch (error) {
+    console.error("프로필 데이터 에러", error);
+    return null;
+  }
```

Comment:
@youngjin-korea 

[제안]

api 호출 함수는 다음과 같이 사용하는게 좋습니다.
에러처리는 호출하는 곳에서 핸들링 합니다. 
음... 예를들어 api 함수를 호출하는 곳이 다양한 페이지에서 사용하게 될텐데요. 그러면 호출하는 쪽에서 상황에 맞게 에러를 핸들링 해야 합니다.

```suggestion
  return await axios
    .get("https://bootcamp-api.codeit.kr/api/sample/user")
    .then(({ data }) => data);
```